Additional 'keycloak' authorization docs (#4743)
strimzi/strimzi-kafka-operator: packaging/examples/security/keycloak-authorization/README.md

# Keycloak authorization example

This folder contains an example of Strimzi custom resource with configured `keycloak` authorization and a corresponding `oauth` authentication,
and a Keycloak realm export that should be imported into your Keycloak instance to support the example.

> Full example instructions are available in [Strimzi Documentation](https://strimzi.io/docs/operators/latest/using.html#con-oauth-authorization-keycloak-example).

Reviewer: ```suggestion
Full example instructions are available in [Strimzi Documentation](https://strimzi.io/docs/operators/in-development/using.html#con-oauth-authorization-keycloak-example).
```

> Full example instructions are available in [Strimzi Documentation](https://strimzi.io/docs/operators/in-development/using.html#con-oauth-authorization-keycloak-example).

Reviewer 2: ```suggestion
Full instructions for the example are available in the [Strimzi Documentation](https://strimzi.io/docs/operators/in-development/using.html#con-oauth-authorization-keycloak-example).
```

* [kafka-authz-realm.json](./kafka-authz-realm.json)
    * The Keycloak realm export file
* [kafka-ephemeral-oauth-single-keycloak-authz.yaml](./kafka-ephemeral-oauth-single-keycloak-authz.yaml)
    * The Kafka CR that defines a single-node Kafka cluster with `oauth` authentication and `keycloak` authorization,

>     using the `kafka-authz` realm. See [full example instructions](https://strimzi.io/docs/operators/latest/using.html#con-oauth-authorization-keycloak-example) for proper preparation and deployment.

Reviewer: ```suggestion
    using the `kafka-authz` realm. See [full example instructions](https://strimzi.io/docs/operators/in-development/using.html#con-oauth-authorization-keycloak-example) for proper preparation and deployment.
```

> * [kafka-authz-realm.json](./kafka-authz-realm.json)
>     * The Keycloak realm export file
> * [kafka-ephemeral-oauth-single-keycloak-authz.yaml](./kafka-ephemeral-oauth-single-keycloak-authz.yaml)
>     * The Kafka CR that defines a single-node Kafka cluster with `oauth` authentication and `keycloak` authorization,
>     using the `kafka-authz` realm. See [full example instructions](https://strimzi.io/docs/operators/in-development/using.html#con-oauth-authorization-keycloak-example) for proper preparation and deployment.

Reviewer 2: ```suggestion
* [kafka-authz-realm.json](./kafka-authz-realm.json)
      The Keycloak realm export file
* [kafka-ephemeral-oauth-single-keycloak-authz.yaml](./kafka-ephemeral-oauth-single-keycloak-authz.yaml)
      The Kafka CR that defines a single-node Kafka cluster with `oauth` authentication and `keycloak` authorization,
    using the `kafka-authz` realm. See [full example instructions](https://strimzi.io/docs/operators/in-development/using.html#con-oauth-authorization-keycloak-example) for proper preparation and deployment.
```

Author: Are you sure about this change? Spacing creates a code block, whereas * creates a bullet point / list item.

Reviewer 2: When I put it in markdown editor, it looks like this:

![image](https://user-images.githubusercontent.com/47596553/119636889-fee15f80-be0c-11eb-97a7-382b1e7864d6.png)
